serverFiles/controllers/articles.controllers.js

```diff
@@ -0,0 +1,17 @@
+const selectArticlesById = require('../models/articles.models');
+
+function getArticlesById(req, res, next) {
+  const { article_id } = req.params;
+  const parsedArticleId = parseInt(article_id);
```

Comment:
Is this necessary? This should already be a number 🤔

Reply:
for some reason i consoled this out and it was always a string. I got rid of the parsing though and put a regex test into my model to check whether it was a digit or not.
I was able to return the correct error code for an invalid type